website/data/docs-nav-data.json

```diff
@@ -403,16 +403,42 @@
         "path": "api-gateway/consul-api-gateway-install"
       },
       {
-        "title": "Technical Specifications",
-        "path": "api-gateway/tech-specs"
+        "title": "Upgrades",
+        "path": "api-gateway/upgrades"
       },
       {
-        "title": "Common Errors",
-        "path": "api-gateway/common-errors"
+        "title": "Usage",
+        "routes": [
+          {
+            "title": "Basic Usage",
+            "path": "api-gateway/usage/basic-usage"
+          }
+        ]
       },
       {
-        "title": "Upgrades",
-        "path": "api-gateway/upgrade-specific-versions"
```

Comment:
Likely needs a redirect added from `api-gateway/upgrade-specific-versions` to `api-gateway/upgrades`?